msi/msi.go

```diff
@@ -8,11 +8,21 @@ import (
 	"fmt"
 	"github.com/Azure/azure-extension-foundation/errorhelper"
 	"github.com/Azure/azure-extension-foundation/httputil"
+	"net/url"
 	"strconv"
 	"time"
 )
 
-const metadataMsiURL = "http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https://management.core.windows.net/"
+const (
+	metadataMsiURL     = "http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https://management.core.windows.net/"
```

Comment:
nit : metadataMsiURL isn't used anywhere, should remove.

Reply:
removed